Grocery_List_Zhecan_Wang/app/src/main/java/com/example/jong/todolist/MainActivity.java

```diff
@@ -0,0 +1,129 @@
+package com.example.jong.todolist;
+
+import android.app.AlertDialog;
+import android.content.DialogInterface;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.util.Log;
+import android.view.Menu;
+import android.view.MenuItem;
+import android.view.View;
+import android.widget.AdapterView;
+import android.widget.ArrayAdapter;
+import android.widget.EditText;
+import android.widget.ListView;
+
+import java.util.ArrayList;
+
+public class MainActivity extends AppCompatActivity {
+    
+    ArrayList<String> arraylist = new ArrayList();
+    ListView listview;
+               
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_main);
+        listview = (ListView) findViewById(R.id.todolist1);
+        listview.setAdapter(new ArrayAdapter<String>(this, R.layout.task_view, arraylist));
+        listview.setOnItemClickListener(new AdapterView.OnItemClickListener() {
```

Comment:
To make your code more separated and more closely follow MVC guidelines you should make a separate implementation of the AdapterView.OnItemClickListener in a different file like we did here.
https://github.com/OlinMobileProto/Lab1/blob/master/app/src/main/java/com/example/cwallace/grocerylist/clickListeners/DeleteEntryListener.java